extensions/ql-vscode/README.md

```diff
@@ -75,23 +77,21 @@ You can find all the commands contributed by the extension in the Command Palett
 
 While you can use the [CodeQL CLI to create your own databases](https://help.semmle.com/codeql/codeql-cli/procedures/create-codeql-database.html), the simplest way to start is by downloading a database from LGTM.com.
 
-1. Log in to LGTM.com.
-1. Find a project you're interested in and display the **Integrations** tab (for example, [Apache Kafka](https://lgtm.com/projects/g/apache/kafka/ci/)).
-1. Scroll to the **CodeQL databases for local analysis** section at the bottom of the page.
-1. Download databases for the languages that you want to explore.
-1. For each database that you want to import:
-    1. Go to the CodeQL Databases view in the sidebar. Hover over the Databases title bar and click the icon to **Choose Database from Archive**.
-    1. Browse to the zipped database that you downloaded from LGTM.
-
-Once you've added a CodeQL database, it is displayed in the Databases view.
+1. Open [https://lgtm.com] in your browser.
+1. Search for a project you are interested in, for example [Apache Kafka](https://lgtm.com/projects/g/apache/kafka/ci/).
```

Comment:
```suggestion
1. Search for a project you're interested in, for example [Apache Kafka](https://lgtm.com/projects/g/apache/kafka).
```

Let's link to the default project tab here, since people no longer need to download databases from the "Integrations" tab!